native/e2e/recorders/android-recorder.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const { rawCommand, command, sleep } = require('./../utils');
+
+function video(filename) {
```

Comment:
I think we have a convention to use the `const video = (filename) => {` format. I prefer using arrow functions myself for the scoping and consistency, but I know you prefer the `function` style. Anybody else have an opinion here?

Reply:
That's a good point, I've copy-pasted that from my library (see PR description) if the arrow function is a deal breaker I'll change it

Reply:
Yeah arrow functions over function declarations please 🙏